fedimint-core/src/config.rs

```diff
@@ -882,26 +880,6 @@ pub enum DkgPeerMsg {
     Encodable(Vec<u8>),
 }
 
-/// Result of running DKG
-pub type DkgResult<T> = Result<T, DkgError>;
-
-#[derive(Error, Debug)]
-/// Captures an error occurring in DKG
-pub enum DkgError {
-    /// User has cancelled the DKG task
-    #[error("Operation cancelled")]
-    Cancelled(#[from] Cancelled),
-    /// Error running DKG
-    #[error("Running DKG failed due to {0}")]
-    Failed(#[from] anyhow::Error),
-    #[error("The module was not found {0}")]
-    ModuleNotFound(ModuleKind),
-    #[error("Params for modules were not found {0:?}")]
-    ParamsNotFound(BTreeSet<ModuleKind>),
-    #[error("Failed to decode encodable message {0:?}")]
-    ModuleDecodeError(DecodeError),
-}
-
```

Comment:
My first reaction was "BUUUUT internationalization", then I realized that none of these errors should happen, even with invalid user input.